src/repositories/transaction.ts

```diff
@@ -35,7 +36,7 @@ export interface ITransactionRepository {
    * @param id    Id for the transaction document that needs to be updated.
    * @param data  Document fields that are about to be updated.
    */
-  updateById(id: string, data: Record<string, any>): Promise<void>
+  updateById(id: string, data: Record<string, unknown>): Promise<void>
```

Comment:
don't we have a Transaction interface defined?

Reply:
same reason as consents